pkgs/applications/science/biology/gatk/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, stdenv, fetchzip }:
+
+# Derivation for GATK
+stdenv.mkDerivation rec {
+  name = "gatk";
+  version = "4.2.6.1";
+  # Rec allows us to use variable in the url
+  src = fetchzip {
+    url = "https://github.com/broadinstitute/gatk/releases/download/${version}/gatk-${version}.zip";
+    sha256 = "0hjlsl7fxf3ankyjidqhwxc70gjh6z4lnjzw6b5fldzb0qvgfvy8";
+  };
+  installPhase =
+    ''
+    install -m755 -D $src/gatk $out/bin/
+    install -m755 -D $src/gatk-package-${version}-local.jar $out/bin/
+    '';
+  meta = with lib; {
+    homepage = "https://gatk.broadinstitute.org/hc/en-us";
+    description = "A wide variety of tools with a primary focus on variant discovery and genotyping." ;
+    license = licenses.asl20;
+    sourceProvenance = with lib.sourceTypes; [ binaryBytecode ] ;
+    longDescription =
+    ''
+    The GATK is the industry standard for identifying SNPs and indels in germline
+    DNA and RNAseq data. Its scope is now expanding to include somatic short variant
+    calling, and to tackle copy number (CNV) and structural variation (SV). In
+    addition to the variant callers themselves, the GATK also includes many
+    utilities to perform related tasks such as processing and quality control of
+    high-throughput sequencing data, and bundles the popular Picard toolkit.
+
+    These tools were primarily designed to process exomes and whole genomes
+    generated with Illumina sequencing technology, but they can be adapted to handle
+    a variety of other technologies and experimental designs. And although it was
+    originally developed for human genetics, the GATK has since evolved to handle
+    genome data from any organism, with any level of ploidy.
+    '';
```

Comment:
```suggestion
    longDescription = ''
      The GATK is the industry standard for identifying SNPs and indels in germline
      DNA and RNAseq data. Its scope is now expanding to include somatic short variant
      calling, and to tackle copy number (CNV) and structural variation (SV). In
      addition to the variant callers themselves, the GATK also includes many
      utilities to perform related tasks such as processing and quality control of
      high-throughput sequencing data, and bundles the popular Picard toolkit.

      These tools were primarily designed to process exomes and whole genomes
      generated with Illumina sequencing technology, but they can be adapted to handle
      a variety of other technologies and experimental designs. And although it was
      originally developed for human genetics, the GATK has since evolved to handle
      genome data from any organism, with any level of ploidy.
    '';
```